docker/Dockerfile.full

```diff
@@ -0,0 +1,61 @@
+# The image tagged with the 'full' suffix
+ARG IMAGE_BASE_NAME
+ARG BASE_IMAGE_HASH
+ARG BASE_MITIE_IMAGE_HASH
+
+FROM ${IMAGE_BASE_NAME}:base-mitie-${BASE_MITIE_IMAGE_HASH} as mitie
+
+FROM ${IMAGE_BASE_NAME}:base-${BASE_IMAGE_HASH} as builder
+
+COPY --from=mitie /build/data /build/data
+
+# copy files
+COPY . /build/
+COPY docker/configs/config_pretrained_embeddings_spacy_en_duckling.yml /build/config.yml
+
+# change working directory
+WORKDIR /build
+
+# install dependencies
+RUN python -m venv /opt/venv && \
+  . /opt/venv/bin/activate && \
+  pip install --no-cache-dir -U 'pip<20'
+RUN . /opt/venv/bin/activate && poetry install --extras full --no-dev --no-root --no-interaction
+RUN . /opt/venv/bin/activate && make install-mitie && \
+  poetry build -f wheel -n && \
+  pip install --no-deps dist/*.whl && \
+  rm -rf dist *.egg-info
+
+# make sure we use the virtualenv
+ENV PATH="/opt/venv/bin:$PATH"
+
+# spacy link
+RUN python -m spacy download en_core_web_md && \
+    python -m spacy download de_core_news_sm && \
+    python -m spacy link en_core_web_md en && \
+    python -m spacy link de_core_news_sm de
+
+# start a new build stage
+FROM ${IMAGE_BASE_NAME}:base-${BASE_IMAGE_HASH} as runner
+
+# copy everything from /opt
+COPY --from=builder /opt/venv /opt/venv
+
+# make sure we use the virtualenv
+ENV PATH="/opt/venv/bin:$PATH"
+
+# update permissions & change user to not run as root
+WORKDIR /app
+RUN chgrp -R 0 /app && chmod -R g=u /app
+USER 1001
+
+# Create a volume for temporary data
+VOLUME /tmp
+
+# change shell
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# the entry point
+EXPOSE 5005
+ENTRYPOINT ["rasa"]
```

Comment:
should we use `tini`?

Reply:
Here it's not needed, we don't use a shell to execute the rasa command.

Reply:
makes sense 👍🏻 Thanks for the explanation!